[2020-06-18 22:25:15,753] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: sh_preprensa.save_processed 2020-06-19T02:24:44+00:00 [queued]>
[2020-06-18 22:25:15,832] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: sh_preprensa.save_processed 2020-06-19T02:24:44+00:00 [queued]>
[2020-06-18 22:25:15,832] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 22:25:15,832] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-18 22:25:15,833] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 22:25:15,959] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): save_processed> on 2020-06-19T02:24:44+00:00
[2020-06-18 22:25:15,963] {standard_task_runner.py:53} INFO - Started process 4582 to run task
[2020-06-18 22:25:16,213] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: sh_preprensa.save_processed 2020-06-19T02:24:44+00:00 [running]> strx
[2020-06-18 22:25:16,276] {logging_mixin.py:112} INFO - Saving processed...
[2020-06-18 22:25:16,315] {python_operator.py:114} INFO - Done. Returned value was: integrate_data
[2020-06-18 22:25:16,372] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=sh_preprensa, task_id=save_processed, execution_date=20200619T022444, start_date=20200619T022515, end_date=20200619T022516
[2020-06-18 22:25:25,694] {logging_mixin.py:112} INFO - [2020-06-18 22:25:25,693] {local_task_job.py:103} INFO - Task exited with return code 0
